<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mytcp</class>
 <widget class="QMainWindow" name="mytcp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>757</width>
    <height>676</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>mytcp</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextBrowser" name="edtRecRobot">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>290</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>服务器端口号：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>260</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>服务器ip地址：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSend">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>460</y>
      <width>131</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>260</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>电脑ip地址：</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>370</y>
      <width>761</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>240</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>网络设置</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edtK4Port">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>290</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>3100</string>
    </property>
   </widget>
   <widget class="QPushButton" name="conRobot">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>320</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>连接电脑</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="edtSend">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>410</y>
      <width>291</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>380</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>发送指令：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>10</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>接收传感器指令：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edtK4IP">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>260</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>192.168.1.242</string>
    </property>
   </widget>
   <widget class="QPushButton" name="conK4">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>290</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>连接传感器</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>0</y>
      <width>16</width>
      <height>371</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>380</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>传感器</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>电脑</string>
     </property>
    </item>
   </widget>
   <widget class="QTextBrowser" name="edtRecK4">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>30</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>接收电脑指令：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>240</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>网络设置</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edtRobotIP">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>260</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>192.168.1.1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearK4">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>320</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>清除显示</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>290</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>电脑端口号：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edtRobotPort">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>290</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>8000</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearRobot">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>320</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>清除显示</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ckbHexSend">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>320</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>HEX发送</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ckbHexShow">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>320</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>HEX显示</string>
    </property>
   </widget>
   <widget class="QPushButton" name="autoCon">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>320</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>自动连接</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>757</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
